_rules/iframe-non-empty-accessible-name-cae760.md

```diff
@@ -142,6 +146,14 @@ This `iframe` element has a negative `tabindex` [attribute value][].
 <iframe tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
 ```
 
+#### Inapplicable Example 4
+
+This `iframe` element has a `role` [attribute value][] of `none`.
+
+```html
+<iframe title=" " src="/test-assets/SC4-1-2-frame-doc.html" role="none"> </iframe>
```

Comment:
This feels like a slightly better test of this. This could be passed because the title attribute is used.

```suggestion
<iframe src="/test-assets/SC4-1-2-frame-doc.html" role="none"> </iframe>
```